src/SFML/Graphics/Texture.cpp

```diff
@@ -313,10 +313,8 @@ bool Texture::loadFromImage(const Image& image, const IntRect& area)
 
         // Adjust the rectangle to the size of the image
         IntRect rectangle = area;
-        if (rectangle.left < 0)
-            rectangle.left = 0;
-        if (rectangle.top < 0)
-            rectangle.top = 0;
+        rectangle.left    = std::max(rectangle.left, 0);
+        rectangle.top     = std::max(rectangle.top, 0);
         if (rectangle.left + rectangle.width > width)
```

Comment:
While at it, let's use `std::min` here too.
```c++
rectangle.width = std::min(rectangle.width, width - rectangle.left);
rectangle.height = std::min(rectangle.height, height - rectangle.top);
```

Reply:
Thanks!